--- PageSettingsIo.qml.orig	2024-03-16 15:01:54.989260400 -0700
+++ PageSettingsIo.qml	2024-03-17 16:33:04.087787900 -0700
@@ -20,7 +20,6 @@
 
 	VeQItemSortTableModel {
 		id: digitalModel
-		filterRegExp: "/[1-9]$"
 		model: VeQItemTableModel {
 			uids: [Utils.path(settingsService, "/Settings/DigitalInput")]
 			flags: VeQItemTableModel.AddChildren |
@@ -83,5 +82,12 @@
 			show: haveBluetooth
 			subpage: Component { PageSettingsBleSensors {} }
 		}
+		
+		MbSubMenu {
+			description: qsTr
+			("RemoteGPIO")
+			subpage: Component {
+			PageSettingsRGPIO {} }
+		}
 	}
 }
