--- services.json.source	2024-03-12 16:45:06.279348500 -0700
+++ services.json.edited	2024-03-12 16:45:06.279348500 -0700
@@ -3848,12 +3848,138 @@
         "type": "enum",
         "name": "Venus relay 4 state",
         "enum": {
           "0": "Open",
           "1": "Closed"
         }
+      },
+	  {
+        "path": "/Relay/4/State",
+        "type": "enum",
+        "name": "Venus relay 5 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/5/State",
+        "type": "enum",
+        "name": "Venus relay 6 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/6/State",
+        "type": "enum",
+        "name": "Venus relay 7 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/7/State",
+        "type": "enum",
+        "name": "Venus relay 8 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/8/State",
+        "type": "enum",
+        "name": "Venus relay 9 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/9/State",
+        "type": "enum",
+        "name": "Venus relay 10 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/10/State",
+        "type": "enum",
+        "name": "Venus relay 11 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/11/State",
+        "type": "enum",
+        "name": "Venus relay 12 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/12/State",
+        "type": "enum",
+        "name": "Venus relay 13 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/13/State",
+        "type": "enum",
+        "name": "Venus relay 14 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/14/State",
+        "type": "enum",
+        "name": "Venus relay 15 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/15/State",
+        "type": "enum",
+        "name": "Venus relay 16 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/16/State",
+        "type": "enum",
+        "name": "Venus relay 17 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
+      },
+	  {
+        "path": "/Relay/17/State",
+        "type": "enum",
+        "name": "Venus relay 18 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        }
       }
     ],
     "accharger": [
       {
         "path": "/Relay/0/State",
         "type": "enum",
@@ -6147,12 +6273,152 @@
         "name": "Venus relay 4 state",
         "enum": {
           "0": "Open",
           "1": "Closed"
         },
         "writable": true
+      },
+      {
+        "path": "/Relay/4/State",
+        "type": "enum",
+        "name": "Venus relay 5 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/5/State",
+        "type": "enum",
+        "name": "Venus relay 6 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/6/State",
+        "type": "enum",
+        "name": "Venus relay 7 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/7/State",
+        "type": "enum",
+        "name": "Venus relay 8 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/8/State",
+        "type": "enum",
+        "name": "Venus relay 9 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/9/State",
+        "type": "enum",
+        "name": "Venus relay 10 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/10/State",
+        "type": "enum",
+        "name": "Venus relay 11 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/11/State",
+        "type": "enum",
+        "name": "Venus relay 12 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/12/State",
+        "type": "enum",
+        "name": "Venus relay 13 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/13/State",
+        "type": "enum",
+        "name": "Venus relay 14 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/14/State",
+        "type": "enum",
+        "name": "Venus relay 15 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/15/State",
+        "type": "enum",
+        "name": "Venus relay 16 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/16/State",
+        "type": "enum",
+        "name": "Venus relay 17 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
+      },
+      {
+        "path": "/Relay/17/State",
+        "type": "enum",
+        "name": "Venus relay 18 state",
+        "enum": {
+          "0": "Open",
+          "1": "Closed"
+        },
+        "writable": true
       }
     ],
     "battery": [
       {
         "path": "/Relay/0/State",
         "type": "enum",
